packages/m/ManUp.js.json

```diff
@@ -0,0 +1,23 @@
+{
+	"name": "ManUp.js",
+	"version": "",
+	"description": "ManUp.js is a polyfill to support the Manifest for Web Apps",
+	"filename": "manup.min.js",
+	"repository": {
+		"type": "git",
+		"url": "git://github.com/boyofgreen/ManUp.js.git"
+	},
+	"keywords": ["boyofgreen", "polyfill", "manup", "manupjs"],
+	"author": "Jeff Burtoft <boyofgreen@gmail.com>",
+	"license": "Apache-2.0",
+	"autoupdate": {
+		"source": "git",
+		"target": "git://github.com/boyofgreen/ManUp.js.git",
```

Comment:
This repository has no tags in it, so we cannot pull any versioned files. Releases need to be tagged before we can use this as an autoupdate source.

Reply:
Ahh, yes... Thanks for the info. Is `autoupdate` a compulsory attribute? Because then I could remove that as judging by what ManUp.js is for, I don't think there would be an update anytime soon as HTML meta tags don't change frequently? Else, I guess I have to contact the owner of the repository to add a tag or release.

Also, shouldn't it be saying `no version found on git` instead of `no version found on npm`?

Reply:
Yes, auto-update is a required attribute, otherwise, we would have no way to add versioned files to the library.

(I've tracked the CI wording error in #275)

Reply:
Oh, thanks! Let me try contacting the owner in that case. Will close this PR and the related issue if the owner doesn't respond.

Reply:
Sounds good to me, thank you :)